src/state/stepReducer.ts

```diff
@@ -8,7 +8,7 @@ export const stepReducer = (
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   action: { type: ACTION_TYPE; value: any },
 ): IAnswer => {
-  switch (action.type) {
+  switch (action?.type) {
```

Comment:
So if `type` is undefined what should happen. (I assume it's the `default` case?) Is this a normal code path or an error-only path?

Reply:
I added a comment to hopefully clarify this. 8406a7a82698ab4b7b4a14b36a6e4c88ea953036